src/cares_wrap.cc

```diff
@@ -318,7 +318,7 @@ void cares_wrap_hostent_cpy(struct hostent* dest, struct hostent* src) {
 
   /* copy `h_name` */
   size_t name_size = strlen(src->h_name) + 1;
-  dest->h_name = node::Malloc<char>(name_size);
+  dest->h_name = reinterpret_cast<char*>(node::Malloc(name_size));
```

Comment:
Can you avoid `reinterpret_cast` when a `static_cast` does the job just fine? Alternatively, you could also backport the overloads of `node::Malloc` & similar methods as they are on `master`, that would help with backporting other patches in the future.